tools/cli/app.go

```diff
@@ -59,6 +59,11 @@ func NewCliApp() *cli.App {
 			Usage:  "optional timeout for context of RPC call in seconds",
 			EnvVar: "CADENCE_CONTEXT_TIMEOUT",
 		},
+		cli.StringFlag{
```

Comment:
This is a global flag. Is it expected to have the flag global?

Reply:
https://github.com/uber/cadence/blob/bec009abeb29fc92a564477efdeb33efe5426170/tools/cli/app.go#L44

When you put a flag here, it’s a global flag. Global means you declare it once for all commands of the CLI tool